src/renderers/shared/fiber/ReactFiberClassComponent.js

```diff
@@ -41,7 +41,7 @@ var invariant = require('invariant');
 const isArray = Array.isArray;
 
 module.exports = function(
-  scheduleUpdate : (fiber : Fiber, priorityLevel : PriorityLevel) => void,
+  scheduleUpdate : (fiber : Fiber, priorityLevel : PriorityLevel, callerName : string | null) => void,
```

Comment:
Should do the same in `ReactFiberUpdateQueue` https://github.com/facebook/react/blob/master/src/renderers/shared/fiber/ReactFiberUpdateQueue.js#L217